rmw_connext_cpp/src/rmw_take.cpp

```diff
@@ -195,6 +202,137 @@ _take(
   return RMW_RET_OK;
 }
 
+rmw_ret_t
+_take_sequence(
+  const rmw_subscription_t * subscription,
+  size_t count,
+  rmw_message_sequence_t * message_sequence,
+  rmw_message_info_sequence_t * message_info_sequence,
+  size_t * taken,
+  rmw_subscription_allocation_t * allocation)
+{
+  (void) allocation;
+  if (!subscription) {
+    RMW_SET_ERROR_MSG("subscription handle is null");
```

Comment:
This should be `RMW_RET_INVALID_ARGUMENT` right?

https://github.com/ros2/rmw/blob/974ef929e22569ed811edbbf70ba85718dbcbc02/rmw/include/rmw/ret_types.h#L32-L37